### PR TITLE
Starts process of submitting form & deployment to github pages

### PR DESCRIPTION
**Pull request by iteles**
This PR:
+ Starts submitting the form https://github.com/nelsonic/perfect-day/issues/32
  + It now submits and saves to the spreadsheet locally
  + Needs steps for Ajax and user feedback https://github.com/dwyl/html-form-send-email-via-google-script-without-server#part-two---make-it-look-good-
+ Gets things ready for deployment https://github.com/nelsonic/perfect-day/issues/33

@nelsonic If you could please review and merge, I'd like to get this up asap so I can test the submission works non-locally and also so that I can start pointing the DNS as it takes 24 hours to propagate.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -181,6 +189,9 @@ <h2 class="ma0 codystar f2 f1-ns dark-gray tc pt3"> Gift Registry </h2>
         <p class="m0 f6 tr pr4 white">If you have a questions <a href="mailto:mdc+wedding@adecoy.com" class='white'>drop us an email</a>.</p>
```
Comment (nelsonic) [inline index.html:189]: Did you *create* this email address?

File: index.html [hunk 2]
```diff
@@ -181,6 +189,9 @@ <h2 class="ma0 codystar f2 f1-ns dark-gray tc pt3"> Gift Registry </h2>
...
       </section>
 
+      <script data-cfasync="false" type="text/javascript"
+  src="https://cdn.rawgit.com/dwyl/html-form-send-email-via-google-script-without-server/master/form-submission-handler.js">
```
Comment (nelsonic) [inline index.html:193]: this submission handler _wont_ work.
i.e. let me _know_ when you want to _pair_ on this...

**Review by nelsonic — APPROVED** [review]
@iteles the form _looks_ good but it's not going to _work_ without modifying the submission handler.
Let me know when you have _time_ to pair on this.

File: index.html (hunk 2 above)
Comment (iteles) [reply index.html:193]: Now would be lovely if you had time, this is what I was hoping we could do this morning!

**Pull request merged**